docs-chef-io/content/server/mutual_tls.md

```diff
@@ -0,0 +1,660 @@
++++
+title = "Mutual Tls"
+
+date = 2021-01-05T15:18:55-08:00
+draft = false
+
+[menu]
+  [menu.server]
+    title = "Mutual Tls"
+    identifier = "server/Mutual Tls"
+    parent = "server"
+    weight = 10
++++
+
+## Configuring Mutual TLS for Chef Server
```

Comment:
We should start this document describing what mTLS is and why someone cares about it in the context of the Chef Infra Server

Reply:
I had that in there and took it out because I assumed it was too much like man-splaining. I'll add it back in. Cheers

Reply:
In docs, we just call it "explaining" and it is totally welcome!